observability/opentelemetry/src/main/resources/org/wildfly/extension/opentelemetry/LocalDescriptions.properties

```diff
@@ -0,0 +1,13 @@
+opentelemetry=OpenTelemetry subsystem
+opentelemetry.add=Operation Adds subsystem
+opentelemetry.batch-delay=The interval, in milliseconds, between two consecutive exports. Default is 5000.
+opentelemetry.endpoint=The URL for the exporter endpoint.
+opentelemetry.export-timeout=The maximum allowed time, in milliseconds, to export data.
+opentelemetry.exporter-type=The name of the exporter to use
+opentelemetry.max-export-batch-size=The maximum size of the export batch.
+opentelemetry.max-queue-size=The maximum size of the queue before it is exported.
```

Comment:
What is the relationship between the queue and batch? Is a batch one unit in the queue?

Reply:
I believe that language is an almost direct copy from the otel docs, but I've tried to make it clearer here. Will push changes momentarily.